package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@superflows/chat-ui-react",
-  "version": "1.1.3",
+  "version": "1.1.4",
```

Comment:
I am sehr confused because this is already 1.1.4 according to main - think this needs a rebase! 😄